### ISSUE TITLE
Hide alias from public space creation (and automatically create)

### ISSUE THREAD

**Issue by niquewoodhouse** [vector-im/element-web#19402]
#### What would you like to do?

When I am making a public space I want to do as little work as possible.  Any work that I do, is work I want to be sure is valuable to making a good space - having a nice image, description, good quality rooms etc.  

#### Why would you like to do it?

I don't like creative tasks feeling like admin, and I feel like creating a public space should feel like an easy, creative task.  I just want my space to be good and I don't believe the alias is relevant to that.  If it's myspecialspace:homeserver.com or myspecialspace1234:homeserver.com makes no real difference.  If it really does matter to me one day I'd probably use a redirecting service or something to make a shorter/prettier URL.

#### How would you like to achieve it?

I'd like to not see an alias input and just have an alias assigned to my space in the background.  This is how I think workspaces work in Slack, I create a workspace by naming it and when I invite people they get an email that has something like this.

<img src="https://user-images.githubusercontent.com/1587166/137353467-2463a661-092f-43fd-b944-dba44efc8482.png" width="400">

### Have you considered any alternatives?

To not have an alias set at all but it does seem like they're very helpful for public spaces.

**Comment by t3chguy** [comment]
Also keep in mind the UX for adding/editing aliases is awful atm, aliases are more than just inbound links though, they also are used for pills to that room/space when used in messages

**Comment by SimonBrandner** [comment]
> they also are used for pills to that room/space when used in messages

That's not true for EleWeb anymore though and it shouldn't be for EleAndroid either, imo

**Comment by kittykat** [comment]
I think this can be closed now as the alias is auto-generated based on the name of the Space and can be over-written by anyone who wants to do that. Assigning to @niquewoodhouse for a decision when he's back

**Comment by turt2live** [comment]
Closing as presumed fixed.

**Comment by t3chguy** [comment]
It is not.

<img width="497" alt="image" src="https://user-images.githubusercontent.com/2403652/173519125-dd200814-e24c-40ae-a707-f970a5cabb1e.png">
